tools/scripts-config-react-webpack/config/webpack.config.umd.js

```diff
@@ -91,8 +95,16 @@ module.exports = options => {
 						use: [
 							{ loader: MiniCssExtractPlugin.loader },
 							{ loader: 'css-loader' },
-							{ loader: 'postcss-loader', options: { plugins: () => [autoprefixer()] } },
-							{ loader: 'resolve-url-loader' },
+							// {
+							// 	loader: 'postcss-loader',
+							// 	options: {
+							// 		postcssOptions: {
+							// 			plugins: ['autoprefixer'],
+							// 		},
+							// 		sourceMap: true,
+							// 	},
+							// },
+							// { loader: 'resolve-url-loader', options: { sourceMap: true } },
```

Comment:
maybe we are missing from some css polyfill here?

Reply:
I have re-aligned with the app configuration